packages/asset-service/package.json

```diff
@@ -30,6 +30,7 @@
     "type-check": "tsc --project ./tsconfig.build.json --noEmit"
   },
   "dependencies": {
+    "@shapeshiftoss/caip": "^6.2.0",
```

Comment:
web should be controlling dependencies at this time with our usage of peer dependencies
```suggestion
```